lib/js/jsffi.nim

```diff
@@ -180,6 +180,7 @@ proc `>`  *(x, y: JsObject): JsObject {.importcpp: "(# > #)".}
 proc `<`  *(x, y: JsObject): JsObject {.importcpp: "(# < #)".}
 proc `>=` *(x, y: JsObject): JsObject {.importcpp: "(# >= #)".}
 proc `<=` *(x, y: JsObject): JsObject {.importcpp: "(# <= #)".}
+proc `**` *(x, y: JsObject): JsObject {.importcpp: "((#) ** #)".}
```

Comment:
pre-existing but why are all these JsObject?
this makes it much less useful.
```nim
when true:
  import jsffi
  proc `**`*[T: SomeNumber](x, y: T): T {.importcpp: "((#) ** #)".}
  doAssert 2**3 == 8
  doAssert 2.0**3 == 8.0
```
can be dealt with as a followup PR though

Reply:
That would be worse, Nim doesn't use `**` as the exponentiation operator.

Reply:
here's the original PR form @juancarlospaco : https://github.com/nim-lang/Nim/pull/12455

my comment remains; as @andreaferretti said here:
https://github.com/nim-lang/Nim/pull/12455#discussion_r347258066
> I don't understand this feedback. Exponentiation does not work on JsObject. In javascript you do not exponentiate two objects, you exponentiate two numbers.